moped-editor/src/views/projects/projectView/ProjectComponents/utils/form.js

```diff
@@ -101,14 +101,35 @@ export const useSubphaseOptions = (subphases) =>
     return options;
   }, [subphases]);
 
+/**
+ * Take the moped_component_tags records data response and create options for a MUI autocomplete
+ * @param {Object} data Data returned with moped_component_tags records
+ * @returns {Array} The options with value, label, and full data object to produce the component tags options
+ */
+export const useComponentTagsOptions = (data) =>
+  useMemo(() => {
+    if (!data) return [];
+
+    const options = data.moped_component_tags.map((tag) => ({
+      value: tag.id,
+      label: `${tag.type.charAt(0).toUpperCase() + tag.type.slice(1)} - ${
```

Comment:
i like what you did here—but i also think we should fix this at the DB level. would you nix this the case handling and i'll fix the "type" values? i'm going to open a migration for that momentarily now that i have new tag values from NW.

Reply:
as promised: https://github.com/cityofaustin/atd-moped/pull/1051